docker/Dockerfile_offline

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 4Paradigm
```

Comment:
We can rename this as "openmldb-build.dockerfile".

Reply:
`full-offline.Dockerfile` still necessary, or it confuse to the original Dockerfile